gradle/libs.versions.toml

```diff
@@ -14,6 +16,10 @@ splitties = "3.0.0-beta01"
 [libraries]
 android-gradle = { module = "com.android.tools.build:gradle", version.ref = "android-gradle-plugin" }
 androidx-annotation = { module = "androidx.annotation:annotation", version.ref = "androidx-annotation" }
+antlr-kotlin-gradle = { module = "com.strumenta.antlr-kotlin:antlr-kotlin-gradle-plugin", version.ref = "antlr-kotlin" }
+#antlr-kotlin-runtime = { module = "com.strumenta.antlr-kotlin:antlr-kotlin-runtime", version.ref = "antlr-kotlin" }
+antlr-kotlin-runtime = { module = "org.cru.mobile.fork.antlr-kotlin:antlr-kotlin-runtime", version.ref = "antlr-kotlin-fork" }
```

Comment:
Do we have our own fork version of this just in case any breaking changes come, it doesn't affect our forked version and therefore doesn't break things?

Reply:
the version currently published on jitpack does not have iOS support, our forked version includes iOS support.